pytential/muller.py

```diff
@@ -55,7 +55,7 @@ def f_deflated(z: T) -> T:
     # Truncates the zero arrays created above if necessary.
     for i in range(n):
         miter = 0
-        roots0, niter0 = muller(f_deflated, maxiter=maxiter, tol=eps)
+        roots0, niter0 = muller(f_deflated, maxiter=maxiter, tol=eps, z_start=z_start)
```

Comment:
For what it's worth, this will also remove the failures on the CI. Basically
* Before, we used `np.random.seed` to set the seed globally so this call always initialized `z_start` to the same random value.
* Now with the change to `np.random.Generator`, this call without `z_start` was essentially unseeded, so it would sometimes hit a value that failed.

Reply:
Maybe this is enough? From a nice blast from the past:
https://gitlab.tiker.net/inducer/pytential/-/issues/26

Reply:
Yeah, let's just pick a good seed and move on.

Reply:
Done in [70c7ed8](https://github.com/inducer/pytential/pull/231/commits/70c7ed842ddeed03dab90aba2c613419481ac72f).

Reply:
Thanks!